Content.Shared/Speech/Components/MeleeSpeechComponent.cs

```diff
@@ -7,15 +8,28 @@ namespace Content.Shared.Speech.Components;
 [AutoGenerateComponentState]
 public sealed partial class MeleeSpeechComponent : Component
 {
+    [ViewVariables]
```

Comment:
Automatic component states don't actually work if the component isn't networked.